astropy/io/fits/convenience.py

```diff
@@ -408,9 +410,11 @@ def writeto(filename, data, header=None, output_verify='exception',
         ``+warn``, or ``+exception" (e.g. ``"fix+warn"``).  See :ref:`verify`
         for more info.
 
-    clobber : bool, optional
-        If `True`, and if filename already exists, it will overwrite
-        the file.  Default is `False`.
+    overwrite : bool, optional
+        If ``True``, overwrite the output file if exists.
```

Comment:
About the `OSError`: shouldn't this be under a separate header `Raises`? Or would that be considered when the function itself raises the error (here the functionality is just passed on,  all the way up to `astropy.io.fits.file`)?

The actual exception raised differs, since [Python 3.3 merged `IOError`](https://docs.python.org/3/whatsnew/3.3.html#pep-3151-reworking-the-os-and-io-exception-hierarchy) and a few other exceptions into `OSError`, and `astropy.io.file._overwrite_existing` explicitly raises an `IOError`.
Perhaps this should be changed to raising an `OSError` (which for people using 2.7 and catching that error, makes that a backward incompatible change).
Otherwise, the docstring could be amended to include the 2.7 case: "raises an ``OSError`` (``IOError`` on Python 2) if the file exists and ``overwrite`` is ``False``."
(And arguably, Python 3 could raise `FileExistsError` instead.)

I don't understand the remark about `io.fits.Conf`: I don't see anything related to overwriting files there.
I would think it awkward design as well, for a keyword with explicit `True` or `False` values (having a default of `None` could trigger such default behaviour, but the default is `False` everywhere afaict).
I'm probably misreading this.